app/views/interactions/index.html.erb

```diff
@@ -29,12 +29,12 @@
           </td>
           <td class="name">
             <%= interaction.label %>
-            <p><%= display_link_for(interaction) %></p>
+            <p><%= link = display_link_for(interaction) %></p>
           </td>
           <td>
           </td>
           <td class='text-center'>
-            <%= link_to('Edit link', edit_local_authority_service_interaction_links_path(interaction_slug: interaction.slug), class: "btn btn-default btn-s") %>
+            <%= link_to(link == 'No link' ? 'Add link' : 'Edit link', edit_local_authority_service_interaction_links_path(interaction_slug: interaction.slug), class: "btn btn-default btn-s") %>
```

Comment:
We went down the route of a helper method similar to display_link_for but essentially felt that one call to a method doing a database lookup would be better than doing two were to actually have a second helper method